manage

```diff
@@ -220,41 +220,41 @@ function initEnv() {
 buildImages() {
   args=${@}
 
-  echo Agents to build: ${BUILD_AGENTS}
-
-  for agent in ${BUILD_AGENTS}; do
-    export BACKCHANNEL_FOLDER=$(dirname "$(find aries-backchannels -name *.${agent})" )
-    echo Backchannel Folder: ${BACKCHANNEL_FOLDER}
-    if [ -e "${BACKCHANNEL_FOLDER}/Dockerfile.${agent}" ]; then
-      echo "Building ${agent}-agent-backchannel ..."
-      local REPO_ARGS
-      REPO_ARGS=
-      if [[ -f "${BACKCHANNEL_FOLDER}/${agent}.repoenv" ]]; then
-        source "${BACKCHANNEL_FOLDER}/${agent}.repoenv"
-        if [[ -n "${REPO_URL}" ]]; then
-          local REPO_COMMIT
-          if [[ -z ${REPO_BRANCH} ]]; then
-            REPO_BRANCH=HEAD
-          fi
-          REPO_COMMIT=$(git ls-remote ${REPO_URL} ${REPO_BRANCH} | cut -f1)
-          REPO_ARGS="--build-arg REPO_URL=${REPO_URL} --build-arg REPO_COMMIT=${REPO_COMMIT}"
-        fi
-      fi
-
-      if ! docker build \
-        ${args} \
-        $(initDockerBuildArgs) \
-        ${REPO_ARGS} \
-        -t "${agent}-agent-backchannel" \
-        -f "${BACKCHANNEL_FOLDER}/Dockerfile.${agent}" "aries-backchannels/"; then
-          echo "Docker image build failed."
-          exit 1
-      fi
-    else
-      echo "Unable to find Dockerfile to build agent: ${agent}"
-      echo "Must be one one of: ${VALID_AGENTS}"
-    fi
-  done
+#   echo Agents to build: ${BUILD_AGENTS}
+# 
+#   for agent in ${BUILD_AGENTS}; do
+#     export BACKCHANNEL_FOLDER=$(dirname "$(find aries-backchannels -name *.${agent})" )
+#     echo Backchannel Folder: ${BACKCHANNEL_FOLDER}
+#     if [ -e "${BACKCHANNEL_FOLDER}/Dockerfile.${agent}" ]; then
+#       echo "Building ${agent}-agent-backchannel ..."
+#       local REPO_ARGS
+#       REPO_ARGS=
+#       if [[ -f "${BACKCHANNEL_FOLDER}/${agent}.repoenv" ]]; then
+#         source "${BACKCHANNEL_FOLDER}/${agent}.repoenv"
+#         if [[ -n "${REPO_URL}" ]]; then
+#           local REPO_COMMIT
+#           if [[ -z ${REPO_BRANCH} ]]; then
+#             REPO_BRANCH=HEAD
+#           fi
+#           REPO_COMMIT=$(git ls-remote ${REPO_URL} ${REPO_BRANCH} | cut -f1)
+#           REPO_ARGS="--build-arg REPO_URL=${REPO_URL} --build-arg REPO_COMMIT=${REPO_COMMIT}"
+#         fi
+#       fi
+# 
+#       if ! docker build \
+#         ${args} \
+#         $(initDockerBuildArgs) \
+#         ${REPO_ARGS} \
+#         -t "${agent}-agent-backchannel" \
+#         -f "${BACKCHANNEL_FOLDER}/Dockerfile.${agent}" "aries-backchannels/"; then
+#           echo "Docker image build failed."
+#           exit 1
+#       fi
+#     else
+#       echo "Unable to find Dockerfile to build agent: ${agent}"
+#       echo "Must be one one of: ${VALID_AGENTS}"
+#     fi
+#   done
```

Comment:
I'm guessing that this change is not intentional?  Did you mean to comment out the building process?

I would like it to not default to building all agents when you don't specify at least one...

Reply:
These changes were commited by mistake, apologies :)